config/kubernetes/config.go

```diff
@@ -31,6 +32,22 @@ import (
 func Configure(p *config.Provider) {
 	p.AddResourceConfigurator("azurerm_kubernetes_cluster", func(r *config.Resource) {
 		r.Version = common.VersionV1Alpha2
+		r.TerraformResource.Schema["kube_admin_config"].Elem.(*schema.Resource).
```

Comment:
nit: We may consider documenting why we need to explicitly mark these as sensitive with code comments. If we have a related issue, we may also refer to that issue.